### PR TITLE
feat: add mobile:viewportRect and mobile:deviceScreenInfo (will be alternative of 'appium:includeDeviceCapsToSessionInfo': false)

### PR DESCRIPTION
**Pull request by KazuCocoa**
I haven't tested mobile commands yet, but probably this works. (Will test later, then update README)
Warning message for `shouldGetDeviceCaps` may also help for our future removal.

---

Note that we'll make appium:includeDeviceCapsToSessionInfo default `false` and remove.
These mobile commands will be the alternative.

### CODE REVIEW

File: lib/commands/execute.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ extensions.executeMobile = async function executeMobile (mobileCommand, opts = {
     //endregion multiple apps management
 
     viewportScreenshot: 'getViewportScreenshot',
+    viewportRect: 'getViewportRect',
```
Comment (mykola-mokhnach) [inline lib/commands/execute.js:77]: Please add the new methods into readme

File: lib/driver.js [hunk 2]
```diff
@@ -1227,6 +1227,11 @@ class XCUITestDriver extends BaseDriver {
     const shouldGetDeviceCaps = _.isBoolean(this.opts.includeDeviceCapsToSessionInfo)
       ? this.opts.includeDeviceCapsToSessionInfo
       : true; // Backward compatibility
+    if (shouldGetDeviceCaps) {
+      log.warn(`Please call 'mobile:deviceScreenInfo' to get 'pixelRatio' and 'statBarHeight', ` +
```
Comment (mykola-mokhnach) [inline lib/driver.js:1231]: I would rather remove this warning as it is the default behaviour, so it could be confusing for users

Comment (KazuCocoa) [reply lib/driver.js]: ok. i just wondered if we addressed this point...

**Comment by KazuCocoa** [comment]
Confirmed these mobile commands worked.

File: README.md [hunk 3]
```diff
@@ -829,6 +829,29 @@ bundle | string | yes | Bundle ID of the XCTest | 'com.bundle.myapp'
 
 Array of xctests in the test bundle (e.g.: `[ 'XCTesterAppUITests.XCTesterAppUITests/testExample', 'XCTesterAppUITests.XCTesterAppUITests/testLaunchPerformance' ]`)
 
+### mobile: viewportRect
+
+Return the rectangle of viewport.
```
Comment (mykola-mokhnach) [inline README.md:834]: Retrieves the viewport dimensions.

File: README.md [hunk 4]
```diff
@@ -829,6 +829,29 @@ bundle | string | yes | Bundle ID of the XCTest | 'com.bundle.myapp'
...
+The viewport is the device screen size without the status bar.
```
Comment (mykola-mokhnach) [inline README.md:835]: The viewport is the device's screen size with status bar size subtracted if the latter is present/visible

File: README.md [hunk 5]
```diff
@@ -829,6 +829,29 @@ bundle | string | yes | Bundle ID of the XCTest | 'com.bundle.myapp'
...
+
+### mobile: deviceScreenInfo
+
+Get information about screen.
+
+#### Returned Result
+
+The response is like `{"value":{"statusBarSize":{"width":414,"height":48},"scale":2}}`
+
+`statusBarSize` is width and height of the result of [status bar](https://developer.apple.com/documentation/xctest/xcuielementtypequeryprovider/1500428-statusbars). `scale` is [screen scale](https://developer.apple.com/documentation/uikit/uiscreen/1617836-scale).
```
Comment (mykola-mokhnach) [inline README.md:853]: `statusBarSize` contains status bar dimensions

File: README.md [hunk 6]
```diff
@@ -829,6 +829,29 @@ bundle | string | yes | Bundle ID of the XCTest | 'com.bundle.myapp'
...
 Array of xctests in the test bundle (e.g.: `[ 'XCTesterAppUITests.XCTesterAppUITests/testExample', 'XCTesterAppUITests.XCTesterAppUITests/testLaunchPerformance' ]`)
 
+### mobile: viewportRect
+
+Return the rectangle of viewport.
+The viewport is the device screen size without the status bar.
+
+#### Returned Result
+
+The response is like `{"value":{"left":0,"top":96,"width":828,"height":1696}}`.
```
Comment (mykola-mokhnach) [inline README.md:839]: looks like

File: README.md [hunk 7]
```diff
@@ -829,6 +829,29 @@ bundle | string | yes | Bundle ID of the XCTest | 'com.bundle.myapp'
...
+
+`width` and `height` are the screen's width and height.
+
+### mobile: deviceScreenInfo
+
+Get information about screen.
+
+#### Returned Result
+
+The response is like `{"value":{"statusBarSize":{"width":414,"height":48},"scale":2}}`
```
Comment (mykola-mokhnach) [inline README.md:851]: looks like

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**